examples/protocol_specification_ex/output/two_party_negotiation/message.py

```diff
@@ -23,14 +38,57 @@ def __eq__(self, other):
             return False
 
 
+class Ali:
+    """This class represents an instance of Ali."""
+
+    def __init__(self):
+        """Initialise an instance of Ali."""
+        raise NotImplementedError
+
+    def __eq__(self, other):
+        """Compare two Ali instances."""
+        if type(other) is type(self):
+            raise NotImplementedError
+        else:
+            return False
+
+
+class _IOTApp:
+    """This class represents an instance of _IOTApp."""
+
+    def __init__(self):
+        """Initialise an instance of _IOTApp."""
+        raise NotImplementedError
+
+    def __eq__(self, other):
+        """Compare two _IOTApp instances."""
+        if type(other) is type(self):
+            raise NotImplementedError
+        else:
+            return False
+
+
 class TwoPartyNegotiationMessage(Message):
     """A protocol for negotiation over a fixed set of resources involving two parties."""
 
     protocol_id = "two_party_negotiation"
```

Comment:
we changed this, it now needs to be `ProtocolId("author", "two_party_negotiation", "version") where `author` and `version` need to match the config

Reply:
Done